README.md

```diff
@@ -0,0 +1,91 @@
+[![Build Status](https://badge.buildkite.com/9e0e6c88972a3248a0908506d6946624da84e4e18c0870c4d0.svg)](https://buildkite.com/rust-vmm/kvm-ci)
+
+# `kvm` - Rust Bindings and Wrappers Linux's Kernel Virtual Machine APIs
+
+This repository contains the `kvm-bindings` and `kvm-ioctls` crates.
+
+## kvm-ioctls
```

Comment:
If you do this, then the crates.io readme will be empty (unless you update the Cargo.toml of each crate to specify the path to the new readme).

The right approach is to have each crate with their own README.md file, and on top of that, also have a top level readme that links to the individual readmes of the individual crates.

kvm-bindings can still be used without kvm-ioctls, so it doesn't make sense to have the readmes together.

Reply:
This was my idea of merging two READMEs together, I was thinking in a "workspace" way and didn't consider we need to publish it. Sorry for the inconvenience @roypat :(

Reply:
I would propose to leave the readmes as they are in this PR, and just add a top-level readme. Then you can open a separate PR in which you update the kvm-bindings readme to fix the problems you've identified.

Reply:
Argh, sorry, this was my bad. Forgot about the crates.io thing. Will post a follow up!